apps/api/src/server.ts

```diff
@@ -35,6 +36,14 @@ const port = process.env.PORT || 3000;
 const startServer = async () => {
 	const app = express();
 
+	if (process.env.IS_DEV) {
+		app.use(
+			cors({
+				origin: 'http://localhost:5173',
+			}),
+		);
+	}
+
```

Comment:
Just wondering, if we need cors in production as well? 
Also origins might be better as a variable in an ENV file. 
@demariadaniel